conda/recipes/libcugraph/meta.yaml

```diff
@@ -33,6 +33,8 @@ requirements:
     - ucx-py {{ minor_version }}
     - ucx-proc=*=gpu
     - gtest
+    - faiss-proc=*=cuda
+    - libfaiss=1.6.3
```

Comment:
Do these need to also be runtime dependencies (as done by cuML [here](https://github.com/rapidsai/cuml/blob/f1abbeb90522090154cc1de657667162fc2a8628/conda/recipes/libcuml/meta.yaml#L51))?

Also, we seem to be okay without it, but I wonder if we need to think about adding a `host` section as well to make our recipe "cross-capable" (as described [here](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host))?  This would also be more consistent with cuML too.